mythical-creatures/exercises/dragon.js

```diff
@@ -1 +1,38 @@
+function createDragon(name, rider, temperment) {
+  var dragon = {
+    name: name,
+    rider: rider,
+    temperment: temperment,
+    hungry: true,
+    timesEaten: 0
+  }
+  return dragon
+}
 
+function greetRider(dragon) {
+  return `Hi, ${dragon.rider}!`
+}
+
+function eat(dragon) {
+  dragon.timesEaten += 1;
+  if (dragon.timesEaten >= 3) {
+    dragon.hungry = false
+  }
+  return dragon
+}
+
+// Spicy:
```

Comment:
Regarding your spicy label questions: 
Is that ok? 
Definitely! 

Is indicating the spicier tests as being spicy helpful for students?
Unsure. It's probably worth just testing out and seeing how it goes. I kind of like there being an easy way to identify a threshold. Instructors could say something like "working on 'spicy' level mythical creatures is beneficial in preparation for the final assessment". If we don't have that label, we have to be very vague... 
I could see some students self selecting out of it and losing valuable learning experiences. However, I also see some students jumping into it too soon if it's not labeled and struggling unproductively. My vote is: try it and see how it goes.